### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by qingyun-wu**
## Why are these changes needed?

- Bump version to 0.3.2
- Minor change in readme

## Checks

- [x] I've included any doc changes needed for https://autogenhub.github.io/autogen/. See https://autogenhub.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by sonichi — COMMENTED** [review]
Shall we comment out the banner for the NuGet package and recover the # downloads/month banner for pyautogen? Also comment out the news for AutoGen.NET? They can be moved to a new separate repo for .NET.

**Comment by marklysze** [comment]
> Shall we comment out the banner for the NuGet package and recover the # downloads/month banner for pyautogen? Also comment out the news for AutoGen.NET? They can be moved to a new separate repo for .NET.

Agreed on the nuget package and AutoGen.NET commenting out. [DONE]

**Comment by marklysze** [comment]
Are we going back to Aug as the date (instead of Oct)?

**Comment by qingyun-wu** [comment]
I am fine either way. The autogen-ai org was created in Aug, and on Oct we moved here.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![Discord](https://img.shields.io/discord/1153072414184452236?logo=discord&style=flat)](https://discord.gg/pAbnFJrkgZ)
 [![Twitter](https://img.shields.io/twitter/url/https/twitter.com/cloudposse.svg?style=social&label=Follow%20%40Chi_Wang_)](https://x.com/Chi_Wang_)
```
Comment (Hk669) [inline README.md:7]: are we updating the twitter handle too in this PR, if the page is created already. otherwise looks good to me.

**Review by Hk669 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (qingyun-wu) [reply README.md:7]: Will udpate later.

**Pull request merged**